packages/admin-panel/src/VizBuilderApp/components/DataLibrary/component/SelectedOptionWithJsonEditor.js

```diff
@@ -86,7 +89,11 @@ export const SelectedOptionWithJsonEditor = ({
         >
           <DownArrow />
         </DownArrowIconWrapper>
-        <BaseSelectedOption option={basicOption} onRemove={onRemove} />
+        <EditableSelectedOption
```

Comment:
I have a feeling we might wanna make this config driven, through some kind of `supportsTitleEditing` property? Currently we can edit titles of aggregations as well which doesn't quite feel right. What do you think?

Reply:
Yep we can add a feature flag in this case.